.github/ISSUE_TEMPLATE/ideas.yml

```diff
@@ -0,0 +1,28 @@
+name: Ideas
+description: Have a suggestion for a new idea or feature?
+title: "[FEATURE REQUEST] <description here>"
+labels: [enhancement]
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Description of the idea or feature 
+    validations:
+      required: true
+  - type: textarea
+    id: screenshots
+    attributes:
+      label: Add several screenshots or links related to the idea or feature (if applicable)
+    validations:
+      required: false
+  - type: textarea
+    id: context
+    attributes:
+      label: Add more details about the idea or feature
+    validations:
+      required: false
+  - type: markdown
+    attributes:
+      value: |
+        You can also join our Discord community [here](http://discord.eddiehub.org)
```

Comment:
Please can you check the link you added? This is referring to another community. Will update it soon with our own community details. Thanks for your contribution. 😊

Reply:
Oh, that was I mistake. I'll make necessary adjustments. Thank you.